### PR TITLE
Small readme fixes

### PR DESCRIPTION
**Pull request by trepel**
Updated readme based on my understanding (I might be wrong).

The important piece is that removal of white char from that s3 bucket name (bucket names can't contain white characters)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The projects `CREDENTIAL_CONFIG.yml` file contains a list of variables which nee
 
 ## Setup
 
-**NOTE**: This project should be forked and made private before proceeding with the following setup steps.
+**NOTE**: This project should be forked before proceeding with the following setup steps. It may be public since all the sensitive informaion will be encrypted.
```
Comment (pmccarthy) [inline README.md]: Not sure about this one. Yes the majority of data is encrypted but it still doesn't feel right to make a credentials repo public

Comment (trepel) [reply README.md]: Only telling me now?! My cred repo is public :) OK, what about relaxing this a bit. Something like "Even though all the sensitive information is encrypted it is recommended to make the repository private" (feel free to suggest better wording). But I am ok with keeping this stanza as is as well.

Comment (pmccarthy) [reply README.md]: We already mention that the repo should be made private here: https://github.com/integr8ly/tower_dummy_credentials#setup

Comment (trepel) [reply README.md]: Ok, updating the wording. @pmccarthy could you have another look?

**Pull request merged**